### DISCUSSION TITLE
Add DSL

### DISCUSSION THREAD

**Discussion by LaserPhaser** [postmanlabs/postman-flows#102] [Ideas]
It would be amazing If you will add any DSL which could be able to use to describe behaviour of the postman flow. 
It will open a door for community to write tons of extensions on top of it.

**Comment by saswatds** [comment]
Hey, @LaserPhaser Thanks for the feedback. This is on our roadmap and we will try to build this at some point. Currently, we are operating on an ever-changing AST as we refine our implementation. Once we have formalised and standardized that, I will try and expose it through YAML or some DSL.

**Reply by LaserPhaser** [reply]
@saswatds If you need any outsource help don't afraid to ask. I have some background in compilers :)
